File: Java/Curso1/peliculas/domain/Pelicula.java
package mx.com.gm.peliculas.domain;

/**
 *
 * @author nieto
 */
public class Pelicula {
    private String nombre;

    public Pelicula() {
    }

    public Pelicula(String nombre) {
        this.nombre = nombre;
    }

    public String getNombre() {
        return this.nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    @Override
    public String toString() {
        return "Pelicula{" + "nombre=" + nombre + '}';
    }
}

File: Python/Repaso_POO/Ejercicio_catalogo/dominio/Pelicula.py
class Pelicula:
    def __init__(self, nombre:str):
        self.__nombre = nombre

    @property
    def nombre(self) -> str:
        return self.__nombre
    
    @nombre.setter
    def nombre(self, nombre: str):
        self.__nombre = nombre

    def __str__(self):
        return self.__nombre